web/router.ex

```diff
@@ -70,8 +70,10 @@ defmodule Ask.Router do
 
       get "/timezones", TimezoneController, :timezones
       resources "/projects", ProjectController, except: [:new, :edit] do
+        post "/panel_survey", SurveyController, :create_panel_survey
         resources "/folders", FolderController, only: [:create, :index, :delete] do
           post "/set_name", FolderController, :set_name
+          post "/panel_survey", SurveyController, :create_panel_survey
```

Comment:
I would not expect to have a separate post to create panel surveys. I do expect a post to create a new incarnation of a panel survey. But that is another story.